source/installation/install-docker.txt

```diff
@@ -0,0 +1,136 @@
+.. _install-docker:
+
+=================================================
+Install with a Containerized Cluster Using Docker
+=================================================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can install Relational Migrator with Kafka in a containerized environment 
+using `Docker <https://docs.docker.com/engine/>`__. This method configures Kafka 
+to store data locally in Docker containers. This deployment method is ideal for 
+educational purposes. Use this if you are looking to learn and understand how to 
+configure your own multi-server Kafka environment. 
+
+.. note::
+
+   This deployment method is not recommended for production workloads. 
+
+About this Task
+---------------
+
+- Relational Migrator is available port 8080.
+
+- In the Docker container, Relational Migrator is installed to 
+  ``/opt/mongodb-relational-migrator/``.
```

Comment:
These look like "Before you Begin" requirements

Reply:
These are actually not prereq's but more of general information so I will go ahead and leave this under About this Task for now.